src/com/google/enterprise/adaptor/documentum/DocumentumAdaptor.java

```diff
@@ -1435,15 +1434,15 @@ public void getDocContent(Request req, Response resp) throws IOException {
         return;
       }
 
-      IDfPersistentObject dmPersObj;
+      IDfSysObject sysObject;
       if (path.matches(".*:\\p{XDigit}{16}")) {
         String chronicleId = path.substring(path.length() - 16);
         logger.log(Level.FINER, "Chronicle ID: {0}", chronicleId);
-        dmPersObj = dmSession.getObjectByQualification(
+        sysObject = (IDfSysObject) dmSession.getObjectByQualification(
```

Comment:
On master at the moment, we're split 50-39 for no dm- prefix, which is pretty far from consistent. Hungarian is generally discouraged, and I've never seen the point of the prefix, especially when it is used together with abbreviations (also discouraged). So sysObject is doubly better than dmPersObj, and multitudes better than the typo dmSysbObj.